pyvenv.cfg

```diff
@@ -0,0 +1,5 @@
+home = /opt/homebrew/opt/python@3.12/bin
+include-system-site-packages = false
+version = 3.12.4
+executable = /opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/bin/python3.12
+command = /opt/homebrew/opt/python@3.12/bin/python3.12 -m venv /Users/daniel.koh/code/api-examples
```

Comment:
Not really sure, if this file is supposed to be pushed in.

I am guessing this is where user will need to input their own directory of where they want to start their virtual env?